compose/ui/ui/src/desktopMain/kotlin/androidx/compose/ui/awt/ComposeDialog.desktop.kt

```diff
@@ -250,6 +250,9 @@ class ComposeDialog : JDialog {
         get() = composePanel.isWindowTransparent
         set(value) {
             composePanel.isWindowTransparent = value
+
+            // Disable macOS shadow for undecorated windows
```

Comment:
I would add a link to the source of this property: https://developer.apple.com/library/archive/technotes/tn2007/tn2196.html#WINDOW_SHADOW

Only found it after reading the Sasha post, Google doesn't have it in the first results.